lib/resources/example_json/r4_examples_coverage.rb

```diff
@@ -281,5 +282,76 @@ module Resources
         }
       }
     ]
+
+    R4_COVERAGE_ENCOUNTER_CREATE ||= {
+      "resourceType": "Coverage",
+      "status": "active",
+      "subscriber": {
+        "reference": "Patient/1626016",
+      "display": "WOLFE, STEPHANIE T"
+    },
+      "extension": [
+      {
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/coverage-encounter",
+        "valueReference": {
+          "reference": "Encounter/31363178"
+      }
+    }
+    ],
+      "subscriberId": "1626016",
+      "dependent": "1626016",
```

Comment:
can we update these, if they don't have to be the same as the beneficiary id?

Reply:
These ids have been used in all postman calls, so I'm not sure if they are suppose to be different.

Reply:
updated [here](https://github.com/cerner/fhir.cerner.com/commit/f0575498e7ea66b3c22a8540b3ef055db4af122d#diff-398829d7a4d0e757ca54b5f56e50d99aL300)